cron/k8s/controller.yaml

```diff
@@ -52,13 +52,21 @@ spec:
           containers:
           - name: controller
             image: gcr.io/openssf/scorecard-batch-controller:stable
-            args: ["cron/internal/data/projects.csv"]
+            args: ["--config=/etc/scorecard/config.yaml", "cron/internal/data/projects.csv"]
```

Comment:
Since the controller uses `os.Args[1]`, the argument positions should be swapped?

Reply:
The controller uses `reader = localFiles(flag.Args())` where 
> flag.Args returns the non-flag command-line arguments.